### PR TITLE
adding llm helm charts

### PR DESCRIPTION
**Pull request by yosefmih**

### CODE REVIEW

File: addons/porter-agent/charts/loki/values.yaml [hunk 1]
```diff
@@ -1090,13 +1090,6 @@ singleBinary:
   terminationGracePeriodSeconds: 30
   # -- Affinity for single binary pods. Passed through `tpl` and, thus, to be configured as string
   # @default -- Hard node and soft zone anti-affinity
-  affinity: |
-    podAntiAffinity:
-      requiredDuringSchedulingIgnoredDuringExecution:
-        - labelSelector:
-            matchLabels:
-              {{- include "loki.singleBinarySelectorLabels" . | nindent 10 }}
-          topologyKey: kubernetes.io/hostname
```
Comment (d-g-town) [inline addons/porter-agent/charts/loki/values.yaml:1099]: did you mean to include this?

File: addons/hf-llm-models/values.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+vllmImage: "docker.io/vllm/vllm-openai:latest"
+
+huggingFaceToken: <the hugging face token if modelId requires it>
```
Comment (d-g-town) [inline addons/hf-llm-models/values.yaml:3]: should you use `#` for yaml commenting?

File: addons/hf-llm-models/values.yaml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+modelID: "mistralai/Mistral-7B-Instruct-v0.2"
+
+gpuMemoryUtilization: 0.98
+
+maxModelLen: 26016
+
+dType: half
+
+efsFileSystemId: <the efs file system id>
```
Comment (d-g-town) [inline addons/hf-llm-models/values.yaml:15]: same question here

**Comment by d-g-town** [comment]
> #### ️✅ There are no secrets present in this pull request anymore.
> If these secrets were true positive and are still valid, we highly recommend you to revoke them. Once a secret has been leaked into a git repository, you should consider it compromised, even if it was deleted immediately. Find [here](https://docs.gitguardian.com/platform/remediate/remediate-incidents) more information about risks.
> 
> 🦉 [GitGuardian](https://dashboard.gitguardian.com/auth/login/?utm_medium=checkruns&utm_source=github&utm_campaign=cr1) detects secrets in your source code to help developers and security teams secure the modern development process. You are seeing this because you or someone else with access to this repository has authorized GitGuardian to scan your pull request.

Whatever secrets were in here should be revoked

**Review by d-g-town — APPROVED** [review]

**Comment by yosefmih** [comment]
> #### ️✅ There are no secrets present in this pull request anymore.
> If these secrets were true positive and are still valid, we highly recommend you to revoke them. Once a secret has been leaked into a git repository, you should consider it compromised, even if it was deleted immediately. Find [here](https://docs.gitguardian.com/platform/remediate/remediate-incidents) more information about risks.
> 
> 🦉 [GitGuardian](https://dashboard.gitguardian.com/auth/login/?utm_medium=checkruns&utm_source=github&utm_campaign=cr1) detects secrets in your source code to help developers and security teams secure the modern development process. You are seeing this because you or someone else with access to this repository has authorized GitGuardian to scan your pull request.

This was due to a leaked personal testing huggingface token that is now revoked.

**Pull request merged**